frontend/src/Components/AskQuestionPage.js

```diff
@@ -2,12 +2,11 @@ import React from "react";
 import "bootstrap/dist/css/bootstrap.css";
 import Header from './Header';
 import QuestionForm from './QuestionForm'; 
-
+import { Link } from 'react-router-dom';
```

Comment:
Doesn't look like you're using this in this file. It's best practice not to import things unless you're using them

Reply:
Thank you.I deleted the line also checked all components.